### PR TITLE
discount added

### PR DESCRIPTION
**Pull request by BiancaSozzi**

### CODE REVIEW

**Comment by francobett** [comment]
OK! TODO:Modify the Event hard coded set on discount.

File: bundesliga_site/views.py [hunk 1]
```diff
@@ -1 +1,49 @@
...
+@method_decorator(login_required, name='dispatch')
+class HomeView(TemplateView, LoginRequiredMixin):
+
+    """ This is the index view. Here we display all the banners that the user
+    has created """
+
+    template_name = 'index.html'
+
+    def get_context_data(self, **kwargs):
+        # import ipdb; ipdb.set_trace()
```
Comment (PKpacheco) [inline bundesliga_site/views.py]: Remove #ipdb

File: bundesliga_app/views.py [hunk 2]
```diff
@@ -8,6 +9,7 @@
 from bundesliga_app.models import Event
 from django.http import HttpResponseRedirect
 from .utils import get_auth_token
+from .models import MemberType, Discount
```
Comment (PKpacheco) [inline bundesliga_app/views.py]: **Just a tip !!** 
Eventbrite has a pattern to import more than 1 thing.
`from .models import (
    Discount,
    MemberType ,
)
`

File: bundesliga_app/views.py [hunk 3]
```diff
@@ -78,3 +80,31 @@ def get_context_data(self, **kwargs):
...
         return context
+
+
+@method_decorator(login_required, name='dispatch')
+class CreateDiscount(TemplateView, LoginRequiredMixin):
+    template_name = 'organizer/createDiscount.html'
+
+    def post(self, *args, **kwargs):
+        # import ipdb
+        # ipdb.set_trace()
```
Comment (eb-asaiz) [inline bundesliga_app/views.py]: NIT: remove debuggers please

File: bundesliga_site/views.py (hunk 1 above)
Comment (francobett) [reply bundesliga_site/views.py]: Fixed on #17

**Pull request merged**